tenseal/cpp/context/tensealcontext.h

```diff
@@ -12,6 +13,11 @@ namespace tenseal {
 using namespace seal;
 using namespace std;
 
+enum class encryption_type {
+    public_key,
+    symmetric,
+};
+
```

Comment:
In terms of naming, I think we have to choose between private/public and asymmetric/symmetric. Personally, I like the second one, as it doesn't have overlapping terms with the different keys (e.g. public keys)